### ISSUE TITLE
Assorted mini-tasks

### ISSUE THREAD

**Issue by zackkrida** [SandersForPresident/WordPress#3]
Since I don't know what is being worked on, but am pretty enthusiastic about this, I thought I'd throw out some tasks I thought could be useful and see if there's any interest.
- [x] Add CSS minification to gulpfile (using https://www.npmjs.com/package/gulp-minify-css)
- [ ] Set up Analytics gmail account and properties for each state
- [x] move all js to wp_footer for performance and switch to jQuery CDN with local fallback (see [here](https://github.com/corysimmons/boy/blob/master/index.html#L35) for example)
- [x] set up acf json save/load 
- [x] require ACF and other plugins with (https://github.com/TGMPA/TGM-Plugin-Activation)

Again, these are just some small optimizations I thought of while I'm unfamiliar with the state of the project--if anything seems like a good idea let me know and i'll spin it into its own pull request! If there's other, more important tasks to be done don't hesitate to let me know.

**Comment by atticoos** [comment]
Feel free to PR all these things, @zackcote :smile:

**Comment by atticoos** [comment]
> Set up Analytics gmail account and properties for each state

What were you thinking about for this? Would we be able to access analytics for all the states in one place still? And then break it down on a state-by-state basis?

**Comment by zackkrida** [comment]
I think that would probably be the most helpful for reporting — it'd be a lot of data in one place but definitely doable. I've done it on multisite installs of 5-10 sites before and things work nicely. It would also mean that with one click we can update all of the event and pageview tracking on every site at once. Exciting stuff!

**Comment by atticoos** [comment]
Exactly why we want MultiSite :) Easily iterate new features and feedback in one place to all state sites

**Comment by zackkrida** [comment]
Yeah, i'm really pumped for this. Much more exciting than the jewelry and marketing sites I usually work on. I'm loving your php structure so far too :+1:

**Comment by atticoos** [comment]
haha :sweat_smile: 

and thanks. I hate using WordPress when it doesn't follow any programming principles
